test/e2e/network/netpol/kubemanager.go

```diff
@@ -110,16 +113,21 @@ func (k *kubeManager) getPod(ns string, name string) (*v1.Pod, error) {
 }
 
 // probeConnectivity execs into a pod and checks its connectivity to another pod..
-func (k *kubeManager) probeConnectivity(nsFrom string, podFrom string, containerFrom string, addrTo string, protocol v1.Protocol, toPort int) (bool, string, error) {
+func (k *kubeManager) probeConnectivity(nsFrom string, podFrom string, containerFrom string, addrTo string, protocol v1.Protocol, toPort int, timeoutSeconds int) (bool, string, error) {
```

Comment:
It's probably worth making this param list a struct given how long the param list has become

Reply:
https://github.com/kubernetes/kubernetes/issues/102334

Reply:
hola, ok added in the follow on issue hope that works @bowei